src/core/utils/mockServerReponseHelper.ts

```diff
@@ -0,0 +1,19 @@
+import { HttpStatusCode } from "../../enums/mockServerResponse";
```

Comment:
Spelling mistake in file name `mockServerResponseHelper.ts`

Reply:
nice catch!